src/blocks/scratch3_looks.js

```diff
@@ -82,33 +82,33 @@ Scratch3LooksBlocks.prototype.hide = function (args, util) {
  * Matches the behavior of Scratch 2.0 for different types of arguments.
  * @param {!Target} target Target to set costume/backdrop to.
  * @param {Any} requestedCostume Costume requested, e.g., 0, 'name', etc.
- * @param {boolean=} opt_zeroIndex Set to zero-index the requestedCostume.
+ * @param {boolean=} optZeroIndex Set to zero-index the requestedCostume.
```

Comment:
note that `opt_` as a prefix seems to be a Closure/Blockly convention - this will just have us diverge from that, which I'm fine with.